content/docs/tasks/telemetry/distributed-tracing/zipkin/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Zipkin
+description: Learn how to configure the proxies to send tracing requests to Zipkin.
+weight: 10
+keywords: [telemetry,tracing,zipkin,span,port forwarding]
+aliases:
+    - /docs/tasks/zipkin-tracing.html
+---
+
+To learn how Istio handles tracing, visit this task's [overview](../overview/).
+
+## Before you begin
```

Comment:
@mmshaikh88 here's the instructions you were looking for today. I hope this merges soon so that others can find the documentation again